benchclients/python/README.md

```diff
@@ -0,0 +1,3 @@
+# benchclients
+
+A lightweight Python package with clients for working with APIs in the Conbench ecosystem
```

Comment:
This one will be only Python, no? Do we need the intermediate `python/` dir?

Reply:
Yeah, I was thinking about this and wasn't sure. With things that are CLIs (datalogistik, benchconnect) there's definitely no need for versions in other languages, but for things like this, it's _possible_ that in the future we might add a client in another language. Like if an existing benchmarking framework in, I don't know, Ruby wanted to send their results to conbench but not have deps on various Python stuff, there's a space here to stick a Ruby port that other people could use. To be clear I don't foresee any ports, but directories are cheap and I'm cautious.